registration/registration-client/src/main/java/io/mosip/registration/controller/device/GenericBiometricsController.java

```diff
@@ -300,7 +301,9 @@ private void displayBiometric(Modality modality) {
 		applicationLabelBundle = ApplicationContext.getBundle(applicationContext.getApplicationLanguage(), RegistrationConstants.LABELS);
 
 		retryBox.setVisible(!isExceptionPhoto(modality));
-		thresholdBox.setVisible(!isExceptionPhoto(modality));
+		thresholdBox.setVisible(!isExceptionPhoto(modality) && isQualityCheckWithSdkEnabled());
```

Comment:
Is this code tested after disabling "QUALITY_CHECK_WITH_SDK"?
"thresholdBox" is a gridpane and inside that we display mdsScore and SDKScore. if "QUALITY_CHECK_WITH_SDK" is set to false, then the whole "thresholdBox" gridpane is hidden.
Can you pls cross check?

Reply:
Yes, I have tested it. If "QUALITY_CHECK_WITH_SDK" is set to 'N' then it is hidden. The next line of code: `GridPane.setMargin(thresholdBox, new Insets(0, 0, isQualityCheckWithSdkEnabled() ? 50 : 0,0))` adjusts the height of the thresholdBoxdepending on whether or not the presence of the quality sdk.